core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -660,6 +669,24 @@ private SqlLibraryOperators() {
           OperandTypes.TIMESTAMP_INTERVAL)
           .withFunctionType(SqlFunctionCategory.TIMEDATE);
 
+  /** The "TIME_SUB(time_expression, INTERVAL int64_expression date_part)"
+   * function (BigQuery) subtracts int64_expression units of date_part from
```

Comment:
s/INTERVAL int64_expression date_part/interval_expression/

In Calcite the INTERVAL keyword isn't specific to the TIME_SUB function, it's part of the interval expression.

Also Calcite doesn't have an int64 type. (Yeah I know BQ does but we're using Calcite terminology here.)